src/docs/development/accessibility-and-localization/accessibility.md

```diff
@@ -83,11 +91,20 @@ and with the largest font setting selected in iOS accessibility settings.
 
 ## Screen readers
 
-Screen readers ([TalkBack][], [VoiceOver][]) enable visually
-impaired users to get spoken feedback about the contents of the screen.
+For mobile, screen readers ([TalkBack][], [VoiceOver][]) enable visually
+impaired users to get spoken feedback about the contents of the screen. Turn on VoiceOver or TalkBack on your mobile device and navigate around your app. 
```

Comment:
TalkBack and VoiceOver also enable interactions with the UI (via gestures on mobile, and keyboard shortcuts on desktop).

Reply:
Also, please break the line after the period. Thx